tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature

```diff
@@ -152,3 +152,18 @@ Feature: users management
       | Brian |
       | Carol |
     And "Admin" logs out
+
+
+  Scenario: admin creates user
+    When "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the users management page
+    And "Admin" creates the following users
+      | name | fullname    | email                   | password |
+      | max  | Max Testing | maxtesting@owncloud.com | 12345678 |
+    When "Max" logs in
+    Then "Max" should have self info:
+      | key         | value                   |
+      | username    | max                     |
+      | displayname | Max Testing             |
+      | email       | maxtesting@owncloud.com |
```

Comment:
Log out for `Admin` and `Max` is missing 🙂

Reply:
thanks)

Reply:
Admin is still missing the logout, or?